frontend/src/components/contacts.tsx

```diff
@@ -1,136 +1,138 @@
-  const [email, setEmail] = useState<string>("");
-  const [phoneNumber, setPhoneNumber] = useState<string>("");
-  const [notes, setNotes] = useState<string>("");
-  const [errorMessage, setErrorMessage] = useState<string>("");
-  const [showError, setShowError] = useState<boolean>(false);
+  )
+  const [eventDate, setEventDate] = useState<string>('')
+  const [eventType, setEventType] = useState<string>('')
+  const [relationshipType, setRelationshipType] = useState<string>('')
+  const [email, setEmail] = useState<string>('')
+  const [phoneNumber, setPhoneNumber] = useState<string>('')
+  const [notes, setNotes] = useState<string>('')
+  const [errorMessage, setErrorMessage] = useState<string>('')
+  const [showError, setShowError] = useState<boolean>(false)
   // TODO T:110 (FK) Support adding addresses
   // const [address, setAddress] = useState<string>('')
 
   const handleChange =
     (setState: React.Dispatch<React.SetStateAction<string>>) =>
-    (event: React.ChangeEvent<HTMLInputElement>) => {
-      setState(event.target.value);
-    };
-  const handleFirstNameChange = handleChange(setFirstName);
-  const handleLastNameChange = handleChange(setLastName);
-  const handleEmailChange = handleChange(setEmail);
-  const handlePhoneNumberChange = handleChange(setPhoneNumber);
-  const handleNotesChange = handleChange(setNotes);
+      (event: React.ChangeEvent<HTMLInputElement>) => {
+        setState(event.target.value)
+      }
+  const handleFirstNameChange = handleChange(setFirstName)
+  const handleLastNameChange = handleChange(setLastName)
+  const handleEmailChange = handleChange(setEmail)
+  const handlePhoneNumberChange = handleChange(setPhoneNumber)
+  const handleNotesChange = handleChange(setNotes)
   // const handleAddressChange = handleChange(setAddress);
 
-  const addRelationship = () => {
-    if (relationshipType.trim() === "") {
-      setShowError(true);
-      return;
+  const addRelationship = (): void => {
+    if (relationshipType.trim() === '') {
+      setShowError(true)
+      return
     }
     setRelationships((prevRelationships) => [
       ...prevRelationships,
       {
-        type: relationshipType,
-      },
-    ]);
-    setRelationshipType("");
-  };
+        type: relationshipType
+      }
+    ])
+    setRelationshipType('')
+  }
 
   const handleRelationshipChange = (
     index: number,
     field: keyof Relationship,
     value: any
-  ) => {
+  ): void => {
     setRelationships((prevRelationships) =>
       prevRelationships.map((type, i) =>
         i === index ? { ...type, [field]: value } : type
       )
-    );
-  };
+    )
+  }
 
-  const handleRemoveRelationshipType = (index: number) => {
+  const handleRemoveRelationshipType = (index: number): void => {
     setRelationships((prevRelationships) => [
       ...prevRelationships.slice(0, index),
-      ...prevRelationships.slice(index + 1),
-    ]);
-  };
+      ...prevRelationships.slice(index + 1)
```

Comment:
is this code meant to simply remove an entry? look into the `splice` method

Reply:
@fahadakhan2